starter-project-web-vue2/pages/yared/_id.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <div v-show="$auth.loggedIn">
+    <UpdateBlog :blog="blog" />
+      <v-btn
+            style="position: absolute; right: 5%"
```

Comment:
Show these options if only the author is logged in. Other users cannot update others' posts.